tkt-64143: Only capitalize fetch RELEASE if everything is default

skarekrow opened the pull request:
Not a local file directory or custom mirror.

Closes: #626
FreeNAS Ticket: #64143

Commit:
Only capitalize fetch RELEASE if everything is default

Not a local file directory or custom mirror.

Closes: #626
FreeNAS Ticket: #64143

jsegaert commented:
Brandon, there's also one in `iocage create` you may consider removing:
https://github.com/iocage/iocage/blob/master/iocage_cli/create.py#L126

Commit:
Address feedback

Remove the upper, users will need to supply a proper name now. Also make checking if the release is newer best effort, as if the users choice of naming causes issues, it's hard to be sure.

skarekrow commented:
backport

themylogin commented:
backport

bugclerk commented:
backport

bugclerk commented:
backport

bugclerk commented:
backport

bugclerk commented:
backport
